api/src/test/java/org/pmiops/workbench/profile/ProfileServiceTest.java

```diff
@@ -55,12 +92,12 @@
   @MockBean
   private VerifiedInstitutionalAffiliationMapper mockVerifiedInstitutionalAffiliationMapper;
 
-  private static final FakeClock CLOCK = new FakeClock(Instant.parse("2000-01-01T00:00:00.00Z"));
+  @MockBean private VerifiedInstitutionalAffiliationDao mockVerifiedInstitutionalAffiliationDao;
```

Comment:
I often trip up on @MockBean vs @Autowired vs @Import ... do we have a doc?

I have probably asked this before.

Reply:
Eek, yeah it would be great to have some docs on this. I've definitely developed an aesthetic that seems to work reliably for most cases, it'll be good to capture that. Tracked in https://precisionmedicineinitiative.atlassian.net/browse/RW-5159